### PR TITLE
Update figures and legends

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

Adding figure legends to all sections
Revising figure numbers according to recent addition of figures

### Directions for reviewers

- [ ] Check whether all figures have legends that sufficiently describe all aspects of the figures
- [ ] Check whether the figure number cited in the section text refers to the appropriate figures

#### Which areas should receive a particularly close look?

Figure legends

#### Is there anything that you want to discuss further?

No

#### Is the pull request ready for review?

Yes

### CODE REVIEW

File: content/03.combining-datasets.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ Ideally, the structure of the composite dataset is reflects differences in varia
...
+![Combining datasets to increase data for training machine learning models. a) Appropriate methods are required to combine smaller datasets into a composite dataset without loss of information: The left panel shows multiple small rare disease datasets that need to be combined to form a dataset of higher sample size. The color of the samples suggest class or groups present in the datasets. The shape correlates to the identity of the datasets. The middle panel shows methods that may be used to combine the datasets. The right panel shows principle component analysis of the combined datasets to verify proper integration without loss of information. b) Appropriately prepared composite datasets can be used to make training, evaluation, and validation datasets for machine learning: Left panel shows the division of the composite dataset into training dataset and a held-out validation dataset (top). The training set is further divided into training and evaluation datasets for k-fold cross-validation. The panel on the right shows the class composition of the training, evaluation, and held-out validation datasets](content/images/figures/pdfs/figure-1-combining-datasets.png){#fig:1}
```
Comment (jaclyn-taroni) [inline content/03.combining-datasets.md]:
```suggestion
![Combining datasets to increase data for training machine learning models. a) Appropriate methods are required to combine smaller datasets into a composite dataset without loss of information: The left panel shows multiple small rare disease datasets that need to be combined to form a dataset of higher sample size. The color of the samples suggest classes or groups present in the datasets. The shape represents the dataset of origin. The middle panel shows methods that may be used to combine the datasets. The right panel shows Principal Component Analysis of the combined datasets to verify proper integration without loss of information. b) Appropriately prepared composite datasets can be used to make training, evaluation, and validation datasets for machine learning: Left panel shows the division of the composite dataset into training dataset and a held-out validation dataset (top). The training set is further divided into training and evaluation datasets for 4-fold cross-validation. The panel on the right shows the class distribution of the training, evaluation, and held-out validation datasets](content/images/figures/pdfs/figure-1-combining-datasets.png){#fig:1}
```

Comment (jaclyn-taroni) [reply content/03.combining-datasets.md]: I would say, more generally, I'm not sure we're concerned about _loss of information_ in the "without reprocessing" example as much as we're concerned about technical artifacts. You would want to make sure you didn't "overcorrect" after batch correction (e.g., you removed biological variation), but that's not what this figure is about in my opinion.

File: content/06.model-complexity.md [hunk 2]
```diff
@@ -38,4 +38,4 @@ In a study using a variational autoencoder (VAE) (see Box 2: Definitions) for di
...
...nel). Running the same ML model on multiple resampled datasets generates a distribution of values for the importance scores for each feature utilized by the ML model (right panel), b) Cascade Learning: A schematic showing the different steps in a cascade learning approach for identifying rare disease patients from electronic health record data[@pmid:30815073]. c) Regularization: A schematic showing the concept of regularization to selectively learn relevant features. The samples (green and blue circles) in the rare disease dataset on left panel can be represented as a combination of features. Each horizontal bar in the middle panel (training set) represents a feature X sample heatmap for one sample each. The dark and light patches show the value of the features in representing the sample denoted by the colored sample next to each bar. In the held-out validation dataset, for a sample of unknown class (open circle), some features recapitulate the pattern present in training set, while others do not. The right panel shows classification accuracy with and without using regularization during implementation of the ML models on rare disease data.](images/figures/pdfs/figure-3.png){#fig:3}
```
Comment (jaclyn-taroni) [inline content/06.model-complexity.md]: I would say what the outcome of cascade learning is maybe? Accuracy goes up, error goes down...

Comment (jaclyn-taroni) [reply content/06.model-complexity.md]: Also consider expanding on this point.

> The right panel shows classification accuracy with and without using regularization during implementation of the ML models on rare disease data.

 I know that you say "to selectively learn features" in the first sentence describing this panel, but that strikes me as potentially a little bit jargon-y.

Comment (jaclyn-taroni) [inline content/06.model-complexity.md]:
```suggestion
![Strategies to reduce misinterpretation of machine learning model output in rare disease. a) Bootstrapping: Left panel shows a small rare disease dataset, which can be resampled with replacement using bootstrap to form a large resampled dataset (middle panel). Running the same ML model on multiple resampled datasets generates a distribution of values for the importance scores for each feature utilized by the ML model (right panel), b) Cascade Learning: A schematic showing the different steps in a cascade learning approach for identifying rare disease patients from electronic health record data[@pmid:30815073]. c) Regularization: A schematic showing the concept of regularization to selectively learn relevant features. The samples (green and blue circles) in the rare disease dataset on left panel can be represented as a combination of features. Each horizontal bar in the middle panel (training set) represents a feature by sample heatmap for one sample each. In the held-out validation dataset, for a sample of unknown class (open circle), some features recapitulate the pattern present in training set, while others do not. The right panel shows classification accuracy with and without using regularization during implementation of the ML models on rare disease data.](images/figures/pdfs/figure-3.png){#fig:3}
```

I found this sentence to be too much information and that's why I think I got confused when I first read it, but it maybe just needs to be rephrased.

> The dark and light patches show the value of the features in representing the sample denoted by the colored sample next to each bar.

**Review by jaclyn-taroni — COMMENTED** [review]
Looks pretty good! I had some comments about the legends getting newly added here.

File: content/03.combining-datasets.md (hunk 1 above)
Comment (jaybee84) [reply content/03.combining-datasets.md]: I may be missing something, but I would consider 
> You would want to make sure you didn't "overcorrect" after batch correction (e.g., you removed biological variation)

as "loss of information".  :)

Comment (jaybee84) [reply content/03.combining-datasets.md]: also cant believe I wrote "principle component analysis" *facepalm*

Comment (jaclyn-taroni) [reply content/03.combining-datasets.md]: I agree with you about the loss of information being important. My concern is that the figure does not show that in my opinion.

Comment (jaybee84) [reply content/03.combining-datasets.md]: Ah! I see what you mean.

Comment (jaybee84) [reply content/03.combining-datasets.md]: @jaclyn-taroni I modified the fig legend a bit more. Take a look when you have a chance and let me know what you think.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻

**Pull request merged**